levelObjects/Actions.md

```diff
@@ -343,17 +343,27 @@ Control values:
 This change has no parameters (all 12 bytes 0x00) and directly returns to the main menu.
 
 
-#### Change State Type 7: Unknown
-
-    0000  int32      Object index 1
-    0004  int32      Object index 2
-    0008  int32      Unknown
-
-> This entry is only used on level R and Alpha grove to define areas between the two objects. Purpose and effect unknown.
+#### Change State Type 7: Rotate Yaw
 
+    0000  int32      Object index
+	0004  [4]byte    Amount
+	0008  [4]byte    Limit
```

Comment:
Amount and Limit appear to be two fields each:
Amount is the amount and a "once only" flag; Limit is composed of a min and a max limit.